src/main/java/melky/resourcepacks/WidgetOverride.java

```diff
@@ -174,42 +453,318 @@ public enum WidgetOverride
 
 		for (WidgetOverride widgetOverride : values())
 		{
-			builder.put(widgetOverride.scriptId, widgetOverride);
+			for (int id = 0; id < widgetOverride.getScriptId().length; id++)
+			{
+				builder.put(widgetOverride.scriptId[id], widgetOverride);
+			}
 		}
 
 		scriptWidgetOverrides = builder.build();
 	}
 
 
-	static class Script
+	static class ScriptId
 	{
-		static final int GE_OFFERS_INIT = 803;
-		static final int GE_OFFERS_SETUP_INIT = 773;
-		static final int KEYBINDS_INIT = 984;
-		static final int KEYBINDS_OPEN_MENU = 985;
+
 		static final int MAKE_ALL_ITEM_HOVER = 2049;
-		static final int MAKE_ALL_QUANTITY_DRAW = 2058;
 		static final int MAKE_ALL_ITEM_OP = 2044;
+		static final int MAKE_ALL_QUANTITY_DRAW = 2058;
 		static final int ACCOUNT_MANAGEMENT_INIT = 2497;
 		static final int CHATBOX_BACKGROUND = 923;
 		static final int FRIENDS_CHAT_BUILD = 1658;
 		static final int FRIEND_UPDATE = 125;
 		static final int IGNORE_UPDATE = 129;
-		static final int QUEST_TAB_TABS_DRAW = 2800;
+		static final int QUEST_TAB_TABS_DRAW = 2799;
 		static final int XPDROPS_SHOW_PANEL = 997;
 		static final int ACHIEVEMENT_DIARY_ENTRY_REBUILD = 2199;
-		static final int KOUREND_TAB_UPDATE = 1229;
 		static final int MINIGAMES_TAB_REBUILD = 435;
 		static final int AUTOCAST_SETUP = 2098;
 		static final int DEATHKEEP_REDRAW = 974;
-		static final int WORLDSWITCHER_INIT = 747;
+		static final int WORLD_SWITCHER_INIT = 747;
 		static final int WORLDMAP_INIT = 1707;
+		static final int WORLD_MAP_DROPDOWN_INIT = 714;
 		static final int BANK_INIT = 274;
 		static final int BANK_SETTINGS_OPEN = 3276;
-		static final int THINKBOX = 715;
-		static final int OPTIONS_TABS_SWITCH = 529;
-		static final int ZEAH_FAVOUR_INIT = 1203;
 		static final int SETTINGS_SIDE_TABS_INIT = 3909;
-		static final int CHANNEL_TAB_INIT = 4471;
+		static final int CHANNEL_TAB_INIT = 2859;
+
+		//new
+		static final int[] GE_SEARCH_INIT = {
+			ScriptID.GE_ITEM_SEARCH, 3558
```

Comment:
Where does 3558 come from? Hunter's scripts says it's `poh_costumes_buttons_searchop`

Reply:
ye it's for that, it uses the same widget as the GE item search with a different scriptId (price checker also opens the item search but uses same scriptId as the GE)
![costume search](https://github.com/melkypie/resource-packs/assets/126547120/f9e75de9-4219-4bee-9450-2eca54beb4d4)

Reply:
Not sure what is expected because I see it being changed already

![image](https://github.com/melkypie/resource-packs/assets/22979513/cbd49c16-25b9-4afa-a845-9ea48a1286c2)

Reply:
Unless you are meaning the border around t he items?